### PR TITLE
Restart firewalld on 15-SP5

### PR DESCRIPTION
**Pull request by grisu48**
Apply the known workaround to fix not working ip-forwarding on 15-SP5.

- Related ticket: https://bugzilla.suse.com/show_bug.cgi?id=1213811
- Verification run: https://duck-norris.qe.suse.de/tests/13382

### CODE REVIEW

**Comment by mloviska** [comment]
Nice catch, nevertheless as it is a workaround it would make sense that devs would have a look at this.

File: tests/containers/bci_test.pm [hunk 1]
```diff
@@ -65,7 +65,7 @@ sub reset_engines {
     my ($self, $current_engine) = @_;
```
Comment (mloviska) [inline tests/containers/bci_test.pm:65]: I think this makes perfect sense to be moved into a `bci_prepare` rather than keeping it in `bci_test`

Comment (grisu48) [reply tests/containers/bci_test.pm:65]: We can, let's do this in a later step, we need to release the BCI containers in the pipeline first.

Comment (grisu48) [reply tests/containers/bci_test.pm:65]: After looking at it more closely, I think we should leave it in the `bci_test.pm` module, as it stops docker when podman is being tested. We cannot do this globally, as we are testing docker and podman within one test run.

Comment (grisu48) [reply tests/containers/bci_test.pm:65]: It's unfortunately needed in both test runs, but we can move it to a library function.

File: tests/containers/bci_test.pm [hunk 2]
```diff
@@ -65,7 +65,7 @@ sub reset_engines {
...
     my ($version, $sp, $host_distri) = get_os_release;
     my $sp_version = "$version.$sp";
-    if ($sp_version =~ /15.3|15.4/) {
+    if ($sp_version =~ /15.3|15.4|15.5/) {
         # This workaround is only needed in SLE 15-SP3 and 15-SP4 (and Leap 15.3 and 15.4)
```
Comment (pdostal) [inline tests/containers/bci_test.pm:69]: and spř

Comment (grisu48) [reply tests/containers/bci_test.pm:69]: I'm sorry, I don't understand

Comment (mloviska) [reply tests/containers/bci_test.pm:69]: I guess he means that the comment is missing `SP5` :D

Comment (grisu48) [reply tests/containers/bci_test.pm:69]: I thought he means to add `15.6` and the ř is just a typo.

Comment (pdostal) [reply tests/containers/bci_test.pm:69]: I meant SP5 but now I see that it's there. Sorry.

File: tests/containers/bci_test.pm (hunk 1 above)
Comment (grisu48) [reply tests/containers/bci_test.pm:65]: See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17527

**Pull request merged**